src/servers/src/auth.rs

```diff
@@ -37,17 +37,17 @@ pub enum Identity<'a> {
     UserId(Username<'a>, Option<HostOrIp<'a>>),
 }
 
-pub type HashedPwd<'a> = &'a [u8];
+pub type HashedPassword<'a> = &'a [u8];
 pub type Salt<'a> = &'a [u8];
-pub type Pwd<'a> = &'a [u8];
 
 /// Authentication information sent by the client.
 pub enum Password<'a> {
-    PlainText(Pwd<'a>),
-    MysqlNativePwd(HashedPwd<'a>, Salt<'a>),
-    PgMD5(HashedPwd<'a>, Salt<'a>),
+    PlainText(&'a [u8]),
```

Comment:
It named `xxxText`, but why does it take a bytes parameter

Reply:
The type of `xxxtext` is a byte array,  which may cause ambiguity. 

Then I will modify it as follows:
```suggestion
    PlainText(&'a str),
```